misk/web/tabs/config/tslint.json

```diff
@@ -0,0 +1,25 @@
+{
+  "defaultSeverity": "warn",
+  "extends": [
+    "tslint:latest", 
+    "tslint-blueprint", 
+    "tslint-clean-code", 
+    "tslint-config-prettier", 
+    "tslint-consistent-codestyle",
+    "tslint-eslint-rules", 
+    "tslint-immutable",
+    "tslint-react",
+    "tslint-sonarts"
+  ],
+  "jsRules": {},
+  "rules": {
+    // "interface-name": false,
```

Comment:
nit: comment why these are commented? Or nuke ’em?

Reply:
Still ironing out what a good, practical linting config is going forward but will remove those soon